src/ILLink.Tasks/LinkTask.cs

```diff
@@ -85,13 +84,6 @@ public class ILLink : ToolTask
 		public bool UnreachableBodies { set => _unreachableBodies = value; }
 		bool? _unreachableBodies;
 
-		/// <summary>
-		///   Boolean specifying whether to enable clearinitlocals optimization globally.
-		///   Maps to '--enable-opt clearinitlocals' or '--disable-opt clearinitlocals'.
-		/// </summary>
-		public bool ClearInitLocals { set => _clearInitLocals = value; }
```

Comment:
Note that once you remove this, the line in the SDK that sets this is going to fail when the new ILLink task version is brought in:

https://github.com/dotnet/sdk/blob/12d968b2be23e1d581a4a92b915ce0d61e14eb11/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets#L103

Reply:
Like https://github.com/mono/linker/pull/1292, I'll wait for the SDK changes to flow into the linker, to get the integration tests passing.